cognee/modules/graph/utils/get_graph_from_model.py

```diff
@@ -1,8 +1,16 @@
 from datetime import datetime, timezone
+
 from cognee.infrastructure.engine import DataPoint
 from cognee.modules.storage.utils import copy_model
 
-def get_graph_from_model(data_point: DataPoint, include_root = True, added_nodes = {}, added_edges = {}):
+
+def get_graph_from_model(data_point: DataPoint, added_nodes=None, added_edges=None):
+
+    if not added_nodes:
```

Comment:
Is there a difference between this and previous code in how the empty dict is created or function is called?

Reply:
Whaaat? This is the most bizarre thing I heard about any programming language. :D Ok, good to know. I'm not sure that other people know this, so please share it.